pkg/api/install.go

```diff
@@ -46,8 +46,8 @@ func Build(m MetricsGetter, podLister corev1.PodLister, nodeLister corev1.NodeLi
 	node := newNodeMetrics(metrics.Resource("nodemetrics"), m, nodeLister)
 	pod := newPodMetrics(metrics.Resource("podmetrics"), m, podLister)
 	metricsServerResources := map[string]rest.Storage{
-		"nodes": node,
-		"pods":  pod,
+		"nodemetrics": node,
```

Comment:
Hi, @PhilipGough, thanks for your pr, but I don’t think it’s a good idea to modify the api resource name directly.  just modify it here is not good at least. I guess that after the modification, kubectl top will not be able to get metrics.Have you tested it?